hudi-spark-datasource/hudi-spark-common/src/main/java/org/apache/hudi/DataSourceUtils.java

```diff
@@ -284,6 +284,8 @@ public static HiveSyncConfig buildHiveSyncConfig(TypedProperties props, String b
         props.getString(DataSourceWriteOptions.HIVE_PASS_OPT_KEY(), DataSourceWriteOptions.DEFAULT_HIVE_PASS_OPT_VAL());
     hiveSyncConfig.jdbcUrl =
         props.getString(DataSourceWriteOptions.HIVE_URL_OPT_KEY(), DataSourceWriteOptions.DEFAULT_HIVE_URL_OPT_VAL());
+    hiveSyncConfig.hiveMetaStoreUri =
```

Comment:
is this not "HIVE_METASTORE_URI_OPT_VAL()" ?

Reply:
wondering if we have any tests around this. if not, we would have caught this. would be nice if we can have one.